includes/class-smaily-for-cf7-request.php

```diff
@@ -131,4 +133,15 @@ public function post() {
 		}
 		return $response;
 	}
+
+	/**
+	 * Find WordPress, Contact Form and Smaily plugin version info and return as string for user-agent.
+	 *
+	 * @return string
+	 */
+	private function get_version_info_for_useragent() {
+		$wp_useragent = 'WordPress/' . get_bloginfo( 'version' ) . '; ' . get_bloginfo( 'url' );
+		$version_info = '; ContactForm7/' . WPCF7_VERSION . '; smaily-for-contact-form-7/' . SMAILY_FOR_CF7_VERSION;
+		return $wp_useragent . $version_info;
```

Comment:
You don't need to change it, but if you'd like to improve readability of the function you could rewrite it to:
```suggestion
		$wp_useragent = array(
			'WordPress/' . get_bloginfo( 'version' ),
			get_bloginfo( 'url' ),
			'ContactForm7/' . WPCF7_VERSION,
			'smaily-for-contact-form-7/' . SMAILY_FOR_CF7_VERSION,
		);
		return implode('; ', $wp_useragent);
```

This way it is easier to notice which parts belong together and what parts are used to compile the User-Agent header.